tests/swagger/support.py

```diff
@@ -0,0 +1,74 @@
+import unittest
+
+from bravado_core.spec import Spec
+from bravado_core.resource import build_resources
+from bravado_core.response import OutgoingResponse
+from bravado_core.request import IncomingRequest
+
+from ..support import (BaseWebTest, MINIMALIST_BUCKET, MINIMALIST_GROUP,
+                       MINIMALIST_COLLECTION, MINIMALIST_RECORD)
+
+
+app = BaseWebTest().make_app()
+
+spec_dict = app.get('/swagger.json').json
+spec = Spec.from_dict(spec_dict)
+resources = build_resources(spec)
+
+
+class SwaggerTest(BaseWebTest, unittest.TestCase):
+
+    def setUp(self):
+        # FIXME: solve memory issues from generating the spec multiple times
+        self.spec_dict = spec_dict
+        self.spec = spec
+        self.resources = resources
```

Comment:
I've tried deallocating them in the `tearDown()` method and leaving it for garbage collecting but it didn't work. Maybe this works different on pytest, but I guess there should be no problems in sharing the spec across the tests. I can't think of a test case we would need to modify these objects.